dashboard/app/jobs.go

```diff
@@ -303,11 +303,33 @@ func bisectCrashForBug(c context.Context, bugKey *db.Key, managers map[string]bo
 		if crash.ReproSyz == 0 || !managers[crash.Manager] {
 			continue
 		}
+		if ok, err := shouldBisectCrash(c, bug, crash, jobType); err != nil {
+			return nil, nil, err
+		} else if !ok {
+			continue
+		}
 		return crash, crashKeys[ci], nil
 	}
 	return nil, nil, nil
 }
 
+func shouldBisectCrash(c context.Context, bug *Bug, crash *Crash, jobType JobType) (bool, error) {
+	if jobType != JobBisectFix {
+		return true, nil
+	}
+	build, err := loadBuild(c, bug.Namespace, crash.BuildID)
+	if err != nil {
+		return false, err
+	}
+	nsConfig := config.Namespaces[bug.Namespace]
+	for _, repo := range nsConfig.Repos {
+		if repo.Alias == build.KernelRepo && repo.FixBisectionDisabled {
```

Comment:
repo.Alias is an arbitrary string which we use in reporting, it does not have to match URL. Branch is also important, because reports are keyed by URL/Branch, so there can be several repos with the same URL but different branches and strictly saying one branch may be rebased and another is not rebased.
This can be replaced with `return !kernelRepoInfo(build).FixBisectionDisabled, nil`.

Reply:
Thanks! I'll make this change.